docs/faq.rst

```diff
@@ -0,0 +1,27 @@
+FAQ
+===
+
+.. _fallback-endpoint:
+
+Can I register an endpoint that accepts all requests?
+-----------------------------------------------------
+
+The fallback endpoint is the endpoint called when an unrecognized request is
+received. By default, the fallback endpoint simply returns a
+``BadRequestError`` to the caller. This behavior may be overriden by
+registering a raw endpoint with ``TChannel.FALLBACK``.
+
+.. code-block:: python
+
+    from tchannel import TChannel
+
+    server = TChannel(name='myservice')
+
+    @server.register('raw', TChannel.FALLBACK)
```

Comment:
This does seem a bit janky. I could add some hacks in place to say scheme isn't required if you're using fallback but that'll complicate an already complicated API.